tailwind.config.js

```diff
@@ -96,11 +86,7 @@ module.exports = {
       },
       maxWidth: {
         "75vw": "75vw",
-      },
-      maxHeight: {
-        "60vh": "60vh",
-        "75vh": "75vh",
-        "80vh": "80vh",
+        "10xl": "104rem", // Base Tailwind ends at 7xl, 80rem. Increments are by 8rem
```

Comment:
All of the max-heights are unused and all are removed. The diff is a bit deceptive here - the new value 10xl is being added to maxWidth, not maxHeight.

Reply:
Also note my logic for why it's 104. I originally had the in-between values of 88rem and 96rem available too, but since they'd be unused there's no point. The wouldn't slow anything down (Tailwind shakes the tree nicely and doesn't render unused classes) but also, whatever.